log/core_wrapper.go

```diff
@@ -2,12 +2,25 @@ package log
 
 import (
 	"strconv"
+	"time"
 
 	"go.uber.org/zap/zapcore"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/reddit/baseplate.go/internal/thriftint"
 )
 
+var (
+	logLatencySeconds = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "baseplate_log_latency_seconds",
+			Help:    "Latency of log calls",
+			Buckets: []float64{0.000005, 0.00001, 0.00005, 0.0001, 0.0005, 0.001, 0.005, 0.01, 0.05, 0.1, 0.5, 1.0},
```

Comment:
nit: for readability, I'd put them one-per-line, with number grouping, and optionally also comments:
```suggestion
			Buckets: []float64{
				0.000_005,
				0.000_010,
				0.000_050,
				0.000_100,
				0.000_500,
				0.001,
				0.005,
				0.01,
				0.05,
				0.1,
				0.5,
				1.0,
			},
```